### PR TITLE
Fixed sample Code issues found during health checkup/toolkit migration.

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**

### CODE REVIEW

File: samples/bot-teams-authentication/nodejs/bots/teamsBot.js [hunk 1]
```diff
@@ -24,10 +24,17 @@ class TeamsBot extends DialogBot {
...
         });
     }
-
+	
     async handleTeamsSigninVerifyState(context, state) {
+        console.log('Running dialog with signin/verifystate from an Invoke Activity.');
         await this.dialog.run(context, this.dialogState);
     }
+
+    async handleTeamsSigninTokenExchange(context, state) {
```
Comment (ChetanSharma-msft) [inline samples/bot-teams-authentication/nodejs/bots/teamsBot.js:33]: Please add methods summary.

File: samples/meetings-live-code-interview/nodejs/ClientApp/package.json [hunk 2]
```diff
@@ -16,8 +17,8 @@
     "@testing-library/user-event": "^13.5.0",
     "axios": "^0.21.4",
     "cors": "^2.8.5",
-    "has-cors": "^1.1.0",
     "fluid-framework": "~0.59.0",
+    "has-cors": "^1.1.0",
```
Comment (ChetanSharma-msft) [inline samples/meetings-live-code-interview/nodejs/ClientApp/package.json:21]: What is the use of this package?

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Comment by ChetanSharma-msft** [comment]
@Harikrishnan-MSFT - Kindly run build pipeline before merging this PR.

File: samples/bot-teams-authentication/nodejs/bots/teamsBot.js (hunk 1 above)
Comment (Harikrishnan-MSFT) [reply samples/bot-teams-authentication/nodejs/bots/teamsBot.js:39]: Fixed

File: samples/meetings-live-code-interview/nodejs/ClientApp/package.json (hunk 2 above)
Comment (Harikrishnan-MSFT) [reply samples/meetings-live-code-interview/nodejs/ClientApp/package.json:21]: Providing a Connect/Express middleware that can be used to enable.

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -** 

https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5875169136

image

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
**Build Pipeline -**
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5937134061

image

**Pull request merged**